sass/srobo/atoms/forms.scss

```diff
@@ -1,16 +1,19 @@
 .form {
-
-  input[type='email'], input[type='text'], select {
+  input[type="email"],
+  input[type="text"],
+  select,
+  textarea {
     border: 1px solid $sr-dark-grey;
     border-radius: 3px;
     font-size: $charlie;
     padding: 12px;
     background: white;
   }
 
-  input, select {
-
-    &[type='submit'] {
+  input,
+  select,
+  textarea {
```

Comment:
Can a `textarea` or a `select` be `type=submit`? What does that do?
(Should `button` be covered here?)

Reply:
Very good question, it was late and I just added text area to everything. Looking back at it I think you are correct. I didn't add a `textarea` of `type=submit` and I don't think it make senses for `select` to be here either. I agree adding `button` would probably be sensible.